src/applications/vaos/project-cheetah/components/FormLayout.jsx

```diff
@@ -9,12 +9,13 @@ import WarningNotification from '../../components/WarningNotification';
 
 export default function FormLayout({ children }) {
   const location = useLocation();
+  const match = useRouteMatch();
   return (
     <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--8">
       <Breadcrumbs>
-        <Link to="new-covid-19-vaccine-booking">COVID-19 vaccine</Link>
+        <Link to={match.url}>COVID-19 vaccine</Link>
```

Comment:
Switching to this means that the logic around being at the start of the flow works when the flow is set to the `/new-covid-19-vaccine-booking` (which is true when running the app normally) and also when it's set to `/` (which happens when we rendering just the `NewBooking` component in unit tests).

`match.url` is always the root url of the routes within the last `Switch` component (probably a better way of describing that).